database/migrations/2023_05_01_000000_add_order_column_to_blog_article_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddOrderColumnToBlogArticleTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('blog_articles', function (Blueprint $table) {
+            $table->integer('order');
```

Comment:
For consistency, let's add this row before the timestamps instead of after them.

```suggestion
            $table->integer('order')->after('slug');
```